src/block/Fire.php

```diff
@@ -186,6 +192,53 @@ private function burnBlock(Block $block, int $chanceBound) : void{
 		}
 	}
 
+	private function spreadFire() : void{
+		$world = $this->position->getWorld();
+		$difficulty7 = $world->getDifficulty() * 7;
+		$age30 = $this->age + 30;
+
+		for($y = -1; $y <= 4; ++$y){
+			//Higher blocks have a lower chance of catching fire
+			$randomBound = 100 + ($y > 1 ? ($y - 1) * 100 : 0);
```

Comment:
This should use `World::Y_MIN`, or fire may not work correctly after the caves expansion is implemented.